src/corefx/System.Globalization.Native/calendarData.cpp

```diff
@@ -556,11 +556,10 @@ extern "C" int32_t EnumCalendarInfo(EnumCalendarInfoCallback callback,
     {
         case ShortDates:
             // ShortDates to map kShort and kMedium in ICU, but also adding the "yMd"
-            // skeleton as well, as this
-            // closely matches what is used on Windows
-            return InvokeCallbackForDateTimePattern(locale, UDAT_YEAR_NUM_MONTH_DAY_UCHAR, callback, context) &&
-                   InvokeCallbackForDatePattern(locale, UDAT_SHORT, callback, context) &&
-                   InvokeCallbackForDatePattern(locale, UDAT_MEDIUM, callback, context);
+            // skeleton as well, as this closely matches what is used on Windows
+            return InvokeCallbackForDatePattern(locale, UDAT_SHORT, callback, context) &&
+                   InvokeCallbackForDatePattern(locale, UDAT_MEDIUM, callback, context) &&
+                   InvokeCallbackForDateTimePattern(locale, UDAT_YEAR_NUM_MONTH_DAY_UCHAR, callback, context);
         case LongDates:
```

Comment:
Yes, it was intentional. I contemplated removing YEAR_NUM_MONTH_DAY all together, but decided that it won't hurt leaving it in. The first one returned is used in the public API, and if there are any duplicates, they will get trimmed. So having this extra value in shouldn't hurt anything.
I moved it last so the MEDIUM pattern would get returned before it. Since I was contemplating removing the YEAR_NUM_MONTH_DAY, it felt like it shouldn't be ahead of MEDIUM.